src/event.rs

```diff
@@ -47,6 +47,7 @@ pub struct ActionContext<'a, N> {
     pub suppress_chars: &'a mut bool,
     pub last_modifiers: &'a mut ModifiersState,
     pub window_changes: &'a mut WindowChanges,
+    pub child_info: &'a tty::ChildInfo
```

Comment:
`ActionContext` needs a way to get at the child PID for `spawn_new_instance`, and this seemed better than a `mut static`.

Reply:
Okay, I replaced it with an `AtomicIsize` and a `pub fn`, and stripped out the extra plumbing.

Originally I did this thinking it'd be easier to deal with for multiple windows (if we ever do that), but that's probably premature.